README.md

```diff
@@ -41,15 +41,15 @@ pytest
 This is an example of how to run the code where the targetstore is a file / in memory store.
 
 ```bash
-python -m pytravharv -cf ./tests/config/ -n base_test.yml -m memory -o ./test.ttl -v -c ./tests/inputs/63523.ttl ./test.ttl
+python -m travharv -cf ./tests/config/ -n base_test.yml -m memory -o ./test.ttl -v -c ./tests/inputs/63523.ttl ./test.ttl
 ```
 
 #### example 2
 
 This is an example of how to run the code where the targetstore is a SPARQL endpoint.
 
 ```bash
-python -m pytravharv -cf ./tests/config/ -n base_test.yml -m uristore -o ./test.ttl -v -ts http://example.org http://example.org/statements
+python -m travharv -cf ./tests/config/ -n base_test.yml -m uristore -o ./test.ttl -v -ts http://example.org http://example.org/statements
 ```
 
 The uri is can be a graphdb repository.
```

Comment:
that sentence is be can will do having strange as to read it?

#moreverbsalwaysmakeitworkmore